package.json

```diff
@@ -35,7 +35,9 @@
     "react-scripts": "1.0.10",
     "react-tabs": "0.8.3",
     "react-test-renderer": "15.6.1",
-    "uswds": "1.0.0"
+    "uswds": "1.0.0",
+    "jest-cli": "^20.0.4",
```

Comment:
i still don't think this is required - i removed it locally and the app still builds for me and tests pass